spyder/app/tour.py

```diff
@@ -44,6 +44,8 @@
     MAIN_TOP_COLOR = QColor.fromRgb(230, 230, 230)
     MAIN_BG_COLOR = QColor.fromRgb(255, 255, 255)
 
+MAC = sys.platform == 'darwin'

```

Comment:
Maybe a little out of scope, but do we not have a global constant for this? Should we add one? We check for Mac countless different places, so it might make sense to centralize it (as we do for a lot of other such constants) at least for new code going forward. Then its cleaner and if we ever need to adjust the check, we can do it one consistent place instead of 1000.